examples/oauth-example/README.md

```
@@ -0,0 +1,23 @@
+
+## Usage
+
+1. Create an `.env` file in this directory with:
+
+```bash
+IRACING_AUTH_CLIENT=<your-client-id>
+IRACING_AUTH_SECRET=<your-client-secret-if-required>
+PORT=3000
+```
+
+2. Run:
```

markdownlint-cli2 (0.22.1)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)